### DISCUSSION TITLE
An error of 'cannot fit 'int' into an index-sized integer' is encountered when using pyinstaller.

### DISCUSSION THREAD

**Discussion by liyaocn** [pyinstaller/pyinstaller#8854] [Help]
![image](https://github.com/user-attachments/assets/35a1c469-97f2-4657-99de-62d4a9ada11a)

arg of  _get_const_info in dis.py is 1558606398545025440777014702142747663316716942674016990006264006802623701520441109414528912392680385707156310033198253885411468593307421506704853107492323026024864603377794414983795737732114517157911089386633003922464975665007493413

**Comment by rokm** [comment]
What version of python is that? Can you also determine which module was being scanned (i.e., by checking the name in `_safe_import_module`?

**Comment by liyaocn** [comment]
python : 3.10

pyinstaller : 6.9

I modified dis.py   like this













At 2024-10-29 17:55:03, "Rok Mandeljc" ***@***.***> wrote:

What version of python is that? Can you also determine which module was being scanned (i.e., by checking the name in _safe_import_module?

**Reply by rokm** [reply]
If this is python 3.10.**0**, then it is an old python issue (see #6301) that has been fixed in python 3.10.1 and later.
